packages/@ember/-internals/glimmer/lib/helpers/action.ts

```diff
@@ -417,7 +417,7 @@ function makeClosureAction(
   return (...args: any[]) => {
     let payload = { target: self, args, label: '@glimmer/closure-action' };
     return flaggedInstrument('interaction.ember-action', payload, () => {
-      return join(self, fn, ...processArgs(args));
+      return join(self, fn as AnyFn, ...processArgs(args));
```

Comment:
Yeah I think it's valid to push the cast down to here 👍🏼

Reply:
<3